exporter/main.py

```diff
@@ -32,16 +33,23 @@ def export_dataset_tables():
         return ('Dataset has no tables.', 500)
 
     for table in tables:
-        dest_uri = "gs://{}/{}-{}.json".format(export_bucket, dataset_name, table.table_id)
+        # split up county-level tables by state and export those individually
+        export_split_county_tables(bq_client, table, export_bucket)
+
+        # export the full table
+        dest_uri = "gs://{}/{}-{}.json".format(
```

Comment:
Should we change all these weird `.format` strings to `fstrings` while we are here?